types/clevertap/clevertap-tests.ts

```diff
@@ -0,0 +1,191 @@
+import * as CleverTap from 'clevertap';
+
+const clevertap = CleverTap.init('accountId', 'accountPasscode', CleverTap.REGIONS.EUROPE);
+
+const data: CleverTap.UploadData[] = [
+    {
+        type: 'profile',
+        identity: '6264372124',
+        profileData: {
+            favoriteColor: 'green',
+            Age: 30,
+            Phone: '+14155551234',
+            Email: 'peter@foo.com',
+        },
+    },
+    {
+        type: 'event',
+        identity: '6264372124',
+        // ts: Math.floor(new Date().getTime() / 1000),
+        evtName: 'choseNewFavoriteFood',
+        evtData: {
+            value: 'sushi',
+        },
+    },
+];
+
+// upload an array of event/profile records callback style
+clevertap.upload(data, { debug: 1, batchSize: 50 }, (res: any) => {
+    // console.log(res);
+});
+
+// or if you prefer Promises
+clevertap.upload(data, { debug: 1, batchSize: 50 }).then((res: any) => {
+    // console.log(res);
+});
+
+// query for events
+const query1 = {
+    event_name: 'choseNewFavoriteFood',
+    props: [{ name: 'value', operator: 'contains', value: 'piz' }],
+    from: 20210101,
+    to: 20210701,
+};
+
+// callback style
+clevertap.events(query1, { debug: 1, batchSize: 500 }, (res: any) => {
+    // console.log(res);
+});
+
+// or if you prefer Promises
+clevertap.events(query1, { debug: 1, batchSize: 500 }).then((res: any) => {
+    // console.log(res);
```

Comment:
```suggestion
```

Reply:
and other places,
you can do:
```ts
res; // $ExpectType ....
```
instead, but you've typed this as any

Reply:
done. The return type is not consistent hence kept it as any.